### PR TITLE
feat: Add net8 target. Add RuntimeIdentifiers for windows-net8.

### PR DESCRIPTION
**Pull request by LProulx86**
GitHub Issue: #

## Proposed Changes

 - [ ] Bug fix
 - [x] Feature
 - [ ] Code style update (formatting)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build or CI related changes
 - [ ] Documentation content changes
 - [ ] Other, please describe:

## What is the current behavior?

BackButtonManage doesn't support net8

## What is the new behavior?

Add support for net8.

## Impact on version

- [ ] **Major** (Public API was modified.)
  - Public constructs (class, struct, delegate, enum, etc.) were removed or renamed.
  - Public members were removed or renamed.
  - Public method signatures were changed or renamed.
- [x] **Minor** (Public API was extended.)
  - Public constructs, members, or overloads were added.
- [ ] **Patch** (Public API was unchanged.)
  - A bug in behavior was fixed.
  - Documentation was changed.
- [ ] **None** (The library is unchanged.)
  - Only code under the `build` folder was changed.
  - Only code under the `.github` folder was changed.

## Checklist

Please check that your PR fulfills the following requirements:

- [ ] Documentation has been added/updated.
- [ ] Automated Unit / Integration tests for the changes have been added/updated.
- [ ] Updated [BREAKING_CHANGES.md](../BREAKING_CHANGES.md) (if you introduced a breaking change).
- [ ] Your conventional commit are aligned with the **Impact on version** section.

### CODE REVIEW

File: src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj [hunk 1]
```diff
@@ -22,6 +22,10 @@
     <IncludeSymbols>true</IncludeSymbols>
     <SymbolPackageFormat>snupkg</SymbolPackageFormat>
   </PropertyGroup>
+	
+	<PropertyGroup Condition="'$(TargetFramework)'=='net8.0-windows10.0.19041'">
```
Comment (carlh98) [inline src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj:26]: The spacing is incorrect

File: src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj [hunk 2]
```diff
@@ -34,17 +38,18 @@
 		<PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
 	</ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)'=='net7.0-windows10.0.19041'">
+  <ItemGroup Condition="'$(TargetFramework)'=='net7.0-windows10.0.19041' Or '$(TargetFramework)'=='net8.0-windows10.0.19041'">
```
Comment (carlh98) [inline src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj:41]: or

File: README.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿# Chinook.BackButtonManager 
+﻿# Chinook.BackButtonManager
```
Comment (carlh98) [inline README.md:1]: revert changes to readme

File: src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj [hunk 4]
```diff
@@ -20,7 +20,11 @@
     <PublishRepositoryUrl>true</PublishRepositoryUrl>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <IncludeSymbols>true</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>	
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(TargetFramework)'=='net8.0-windows10.0.19041'">
```
Comment (Soap-141) [inline src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj:26]: Is this also needed for .NET 7?

Comment (LProulx86) [reply src/BackButtonManager.Uno.WinUI/BackButtonManager.Uno.WinUI.csproj:26]: It's a breaking change from net8 see :

RuntimeIdentifiers have a new RID format see: [rid-graph](https://learn.microsoft.com/en-gb/dotnet/core/compatibility/sdk/8.0/rid-graph)
ex: win10-x86 is now win-x86

**Review by Soap-141 — APPROVED** [review]

File: BREAKING_CHANGES.md [hunk 5]
```diff
@@ -1,5 +1,8 @@
 # Breaking Changes
 
+## 2.1.0
```
Comment (jeanplevesque) [inline BREAKING_CHANGES.md:3]: This isn't a breaking change.

Comment (Soap-141) [reply BREAKING_CHANGES.md:3]: Think it was added like the other version changes.

**Pull request closed without merging**